### ISSUE TITLE
configure issue on CentOS Linux release 8.1.1911

### ISSUE THREAD

**Issue by wangdbang** [crosstool-ng/crosstool-ng#2093]
crosstool-NG version: 1.26.0
host OS and version: CentOS Linux release 8.1.1911
I configure the crosstool-ng with command ./configure, it reported "unary operator expected", 
"line 12230: test: !=: unary operator expected
line 12236: test: !=: unary operator expected
line 12261: test: =: unary operator expected
line 12265: test: =: unary operator expected"
The configure log is attached.
[cfg.log](https://github.com/crosstool-ng/crosstool-ng/files/14048873/cfg.log)

**Comment by QBos07** [comment]
Are you sure you used bash 10?

**Comment by wangdbang** [comment]
[root@localhost ~]# bash --version
bash --version
GNU bash, version 4.4.19(1)-release (x86_64-redhat-linux-gnu)
Copyright (C) 2016 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>

**Comment by QBos07** [comment]
Bash 4 shoud be enought. I did a bit of research. We support bash 3.1 and up.
Does youre `/bin/sh` point to `bash` or something like `dash` or `ash`?

**Comment by cpackham** [comment]
I assume somewhere we're failing to portably quote something. Unfortunately because `./configure` is generated I can't tell what is on line 12230. I did try spinning up a centos8 container but because it's EoL it didn't go smoothly and I ended up not being able to install the prerequisites needed.

Would it be possible for you to share what is on lines 12230, 12236, 12261 and 12265 of the configure script for you.
